src/features/profile/profileScreenStyles.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import {StyleSheet} from 'react-native';
```

Comment:
This file should have .ts extension. Because we don't use JSX syntax here.
Also we don't need import React here.